File: 02 Algorithm Reference/02 Initializing Algorithms/06 Cash and Brokerage Models/03 code.fs
(* In Initialize method, set the brokerage models *)
this.SetBrokerageModel(BrokerageName.InteractiveBrokersBrokerage, AccountType.Cash);
this.AddEquity("SPY", Resolution.Second) |> ignore
(* Then if required, you can set specific margin models for your secutities *)
this.Securities.Item("SPY").MarginModel <- new PatternDayTradingMarginModel()

File: 02 Algorithm Reference/13 Reality Modelling/02 Brokerage Models/03 code.py
# Set brokerage model using helper methods:
self.SetBrokerageModel(BrokerageName.FxcmBrokerage) # Defaults to margin account
self.SetBrokerageModel(BrokerageName.TradierBrokerage, \ 
                       AccountType.Cash) # Or override account type

# Supported brokerage names:
BrokerageName.FxcmBrokerage
             .OandaBrokerage
             .TradierBrokerage
             .InteractiveBrokersBrokerage